### PR TITLE
700 zone search

### PR DESCRIPTION
**Pull request by JamieRL**
Added search by zone functionality to widget page. Hooked up same as article page etc. Search method is in ThemeManager, passes back to view which handles serialization of objects.

### CODE REVIEW

**Comment by JamieRL** [comment]
Adding tests.

File: dispatch/static/manager/src/js/pages/Widgets/ZoneIndexPage.js [hunk 1]
```diff
@@ -7,10 +7,37 @@ import ItemList from '../../components/ItemList'
...
+      query.q = this.props.location.query.q
+    }
+    return query
+  }
+
+  isNewQuery(prevProps, props) {
+    return prevProps.location.query.q !== props.location.query.q
+  }
+
+  handleSearchZones(query) {
```
Comment (psiemens) [inline dispatch/static/manager/src/js/pages/Widgets/ZoneIndexPage.js]: This method can be taken out -- you can just call `this.props.searchZones` directly

**Review by psiemens — CHANGES_REQUESTED** [review]
This looks really clean. Nice use of the `ThemeManager`. Just one small fix before merging

**Review by psiemens — APPROVED** [review]

**Comment by psiemens** [comment]
Looks good, merge away 👍

**Pull request merged**